xsnippet_api/application.py

```diff
@@ -58,7 +58,8 @@ def create_app(conf):
     # we need to import all the resources in order to evaluate @endpoint
     # decorator, so they can be collected and passed to VersionRouter
     from . import resources  # noqa
-    app = web.Application(router=router.VersionRouter(endpoint.collect()))
+    app = web.Application(router=router.VersionRouter(endpoint.collect()),
+                          middlewares=[middlewares.auth])
```

Comment:
Yeah, I think it's one of those, that will always be enabled. The fact that it's a middleware helps keep code clean and modular, though.